lib/bibliothecary/parsers/swift_pm.rb

```
@@ -30,6 +36,32 @@ def self.parse_package_swift(file_contents, options: {}) # rubocop:disable Lint/
           }
         end
       end
+
+      def self.parse_package_resolved(file_contents, options: {})
+        json = JSON.parse(file_contents)
+        if json["version"] == 1
+          json["object"]["pins"].map do |dependency|
+            name = dependency['repositoryURL'].gsub(/^https?:\/\//, '').gsub(/\.git$/,'')
```

RuboCop (1.75.5)

[convention] Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.

(Style/StringLiterals)